.github/workflows/release_linux_aarch64.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           bash -exc '\
           source .env/bin/activate && \
           python -m pip uninstall -y protobuf numpy && python -m pip install -q -r requirements-release.txt && \
-          python -m pip install -q onnxruntime && \
+          python -m pip install -q onnxruntime==1.17.3 && \
```

Comment:
Possible to move this to a requirement file somewhere so it's always in sync? I suggest https://github.com/onnx/onnx/blob/main/requirements-release.txt

Reply:
We shall remove these tests from release CIs. The test failures are informative but not blocking. All test failures are either unsupported ops or small numerical discrepancies due to numpy verions.